build-info-extractor-docker/src/main/java/org/jfrog/build/extractor/docker/DockerUtils.java

```diff
@@ -226,4 +228,11 @@ public static Boolean isImageVersioned(String imageTag) {
         int indexOfLastColon = imageTag.lastIndexOf(":");
         return indexOfFirstSlash < indexOfLastColon;
     }
+
+    // Set the Java temp dir system property. As a result, java will create it for us.
```

Comment:
Let's add to this description the reason for why we're doing this. Without it, the random reader will have no idea why this is done.